spec/controllers/user_sessions_controller_spec.rb

```diff
@@ -15,7 +15,21 @@
 
       before { get :security_notification }
 
-      it { is_expected.to redirect_to(account_path) }
+      let(:expected_site_path) { site_path(id: current_user.affiliates.first.id) }
+
+      it { is_expected.to redirect_to(expected_site_path) }
+    end
+
+    context 'when a not_approved user is logged in' do
+      include_context 'not_approved user logged in'
+
+      render_views
+
+      before { get :security_notification }
+
+      it 'shows the access denied text' do
+        expect(response.body).to have_content(LandingPageFinder::ACCESS_DENIED_TEXT)
+      end
```

Comment:
We can nix this spec and the associated `'when a not_approved user is logged in'` context, as that is not a valid scenario. We can cover a not_approved user trying and failing to log in via the cucumber test I mentioned. The only other scenario we might want to test is "an approved user logs in, is set to not_approved while they are still logged in, and tries to visit another page", but that could be better handled via a system or cucumber spec (and doesn't need to handled as part of SRCH-947, as it's separate from the landing page behavior).

Reply:
Removed.